app/controllers/api/v1x0/workflows_controller.rb

```diff
@@ -76,12 +77,8 @@ def rbac_scope(relation)
         relation
       end
 
-      def workflow_params
-        params.permit(:object_id, :object_type, :app_name, :name, :description, :group_refs => [])
-      end
-
       def attrs
-        workflow_params.slice(:object_id, :object_type, :app_name).to_unsafe_h
+        params.slice(:object_id, :object_type, :app_name).to_unsafe_h
```

Comment:
@hsong-rh do we need this since the resource object only has those exact 3 attributes
https://github.com/ManageIQ/approval-api/blob/c2725503a8a2f8fa408c5253b82b1e2d6b5158f9/public/doc/openapi-3-v1.0.0.json#L1340
can't we just use params_for_create and it would do the seem thing and we won't need the attrs method